src/app/system/surveys/surveys.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+    <button mat-raised-button color="primary" [routerLink]="['create']">
+      <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+      Create Surveys
+    </button>
+  </div>
+  
+  <div class="container">
+  
+    <div fxLayout="row">
+      <mat-form-field fxFlex>
+        <mat-label>Filter</mat-label>
+        <input matInput (keyup)="applyFilter($event.target.value)">
+      </mat-form-field>
+    </div>
+  
+    <div class="mat-elevation-z8">
+      <table mat-table [dataSource]="dataSource" matSort>
+        <ng-container matColumnDef="key">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header='key'> Key </th>
+            <td mat-cell *matCellDef="let row"> {{row.key}} </td>
+          </ng-container>
+          
+          <!-- Name Column -->
+          <ng-container matColumnDef="name">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header='name'> Name </th>
+            <td mat-cell *matCellDef="let row"> {{row.name}} </td>
+          </ng-container>
+    
+          <!-- Description Column-->
+          <ng-container matColumnDef="description">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header='description'> Description </th>
+            <td mat-cell *matCellDef="let row"> {{row.description}} </td>
+          </ng-container>
+    
+          <!-- CountryCode Column -->
+          <ng-container matColumnDef="countryCode">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header='countryCode'> Country Code </th>
+            <td mat-cell *matCellDef="let row"> {{row.countryCode}} </td>
+          </ng-container>
+    
+    
+          <ng-container matColumnDef="status">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header='status'> Status </th>
+            <td mat-cell *matCellDef="let row">
+                <p mat-raised-button [hidden]="!isActive(row.validFrom, row.validTo)" >Active</p>
```

Comment:
Instead of just showing text here, modify it so it shows an indicator along with a tooltip as it does for the disabled column in the chart of accounts component. The tooltip should display Active/Inactive.
For your reference: https://github.com/openMF/web-app/blob/master/src/app/accounting/chart-of-accounts/chart-of-accounts.component.html